e2e/app/eoa/static.go

```diff
@@ -0,0 +1,176 @@
+package eoa
+
+import (
+	"math/big"
+
+	"github.com/omni-network/omni/lib/anvil"
+	"github.com/omni-network/omni/lib/netconf"
+
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+//nolint:gochecknoglobals // Static mappings.
+var statics = map[netconf.ID][]Account{
+	netconf.Devnet: {
+		{
+			Type:          TypeRemote,
+			Role:          RoleCreate3Deployer,
+			Address:       devnetCreate3Deployer,
+			Chains:        ChainSelectorAll,
+			MinBalance:    big.NewInt(10),
```

Comment:
balancers are in wei, so this is 10 wei, which is basically 0. I suggest you create some globals:
```
var (
  ether1 = new(big.Int).Mul(big.NewInt(1),big.NewInt(params.Ether)
  ether5 = new(big.Int).Mul(big.NewInt(5),big.NewInt(params.Ether) 
  ether10 ...
  etc
)
```

Reply:
Suggest also to group threshold balances into 3 groups maybe:
```
var (
  minBalanceSmall = gwei100
  targetBalanceSmall = gwei500
  
  minBalanceMedium = ether1
  targetBalanceMedium = ether5
  
  minBalanceLarge = ether10
  targetBalanceLarge = ether100 
)
```

then map accounts to one of these. Please confirm these balances in slack thread with the team